app/frontend/src/features/userQueries/useUsers.ts

```diff
@@ -23,10 +23,12 @@ export default function useUsers(ids: number[], invalidate: boolean = false) {
   }, [handleInvalidation]);
 
   //arrays use reference equality, so you can't use ids in useEffect directly
-  if (!arrayEq(idsRef.current, ids)) {
-    idsRef.current = ids;
-    handleInvalidation();
-  }
+  useEffect(() => {
```

Comment:
Oh I see what you mean, yeah let's go with this for better usability of the hook overall 👍